content/tutorials/unityxrpk-networked.mdx

```diff
@@ -0,0 +1,58 @@
+
+We have prepared a sample project for you which you can download [here](https://github.com/PlutoVR/unity-networked-xrpk-sample). The project utilizes [Photon Unity Networking (PUN)](https://id.photonengine.com/connect/authorize?client_id=Live.WWW&redirect_uri=https%3A%2F%2Fwww.photonengine.com%2Fsignin-oidc&response_type=code&scope=openid%20egweb.profile&code_challenge=5DXCoaVehvLCLDM1AzHvkMTWzDgNKl9Q6wBe9Hj9MTY&code_challenge_method=S256&response_mode=form_post&nonce=637622696265917553.ZjdmYzNjOTgtNTRjYi00YjU0LTk2ZjQtMzQ5MDBmYmRhYjRkMDlhYjhjNTItNjAwOC00NWYwLThhMTEtZGQ4ZGE3ODJlNjgx&ui_locales=en-US&state=CfDJ8HsbaHtZH1VIll6ZzG1iqijHciVFs7-kG7n_iOQ9Q9pUiOFCzyy0bO-7ZfC9LGvepggG4MNtfKm1WaOzVx_TZWZlRdGJh4XGGNqJOhxoUE-dujBqZOLP6Cs4MvdTpQ5FHvULAMyf7o2a4jap7KIr1928WyoASmuJB13uLpEQVFe0NvTrvAl4dW_qmJMpxNIrxm_e3kSsvnrib8Bp7rZzKeUak5AEk9jG4H6qWVcopQlg334vaXWp0Jqh0Y34jBu0swaTI-Lmo5XEKJ8_XZAvIVDHsAemAchCgqLEwYNGvBrc8R9K61_RWCFwhlq3mmMX28gBF8FtKPK1prF_glWUHSWZ3vviurISUef88ogtstsd3FgBLm9deyGDXFPn0H77Wg&x-client-SKU=ID_NETSTANDARD2_0&x-client-ver=5.5.0.0),
+a popular networking library for Unity. Luckily, Photon has a free tier, so you can download the project, add your own Photon app ID (which you can get by [registering your app](https://developer.photonengine.com/dashboard/)) and witness XR Package networking in action!
+
+It's important to mention you can use _any_ networking solution you like, as long as it's _supported in the WebGL runtime_ (which unfortunately, as of the time of writing, excludes the equally popular Normcore.)
+
+You'll notice the project is very similar to the Sample Package you've encountered in the previous tutorial. In fact, the only difference is the inclusion and implementation of Photon (which also involves a minor refact to the Cubes prefab - it is now instantiated when the first user has joined, or created, the room).
+
+Before we start, we recommend you acquaint yourself with [Photon/PUN fundamentals](https://doc.photonengine.com/zh-cn/pun/v2/demos-and-tutorials/pun-basics-tutorial/intro).
+It's important to note this is just an example networking implementation - the XR Package-critical step will be described in step 2.
+
+## 0. Preparation
+
+As mentioned, you'll have to add your own App ID to Photon - this is a unique identifier for your app, and is used to identify your app when connecting to a specific Photon server. It's free to get (see above link), and once you have it, you'll need to add it to the project.
+Go to `Window/Photon Unity Networking/Highlight Server Settings"` and add your App ID. You should be good to go - hitting `Play` will demonstrate the stages of connecting to a server in the console and instantiate the Cubes prefab.
+
+## 1. General Scene Changes
+
+a. Note that the Left Controller and Right Controller scene objects now have additional components added to them: a Photon View and a Photon Transform View.
```

Comment:
The 'a' and 'b' subsections don't really stand out, so perhaps they could be placed below ### headers